compiler/rustc_ast/src/token.rs

```diff
@@ -710,7 +710,11 @@ impl Token {
         }
     }
 
-    pub fn glue(&self, joint: &Token) -> Option<Token> {
+    /// Checks if the current token is a multiple-character token (or glued like `==`, `<=`, `...`)
+    /// and returns the corresponding `TokenKind` if it is.
+    ///
+    /// if the current tokens is already complete, returns `None`.
```

Comment:
I don't think this comment explains what this method is doing properly.

What this is doing is checks if the current token can be combined with the `joint` token to turn into a multiple-character token.